digitaloceanmodule.php

```diff
@@ -430,7 +430,7 @@ public function getSizesDropdown($module_row)
         $dp = array();
         foreach ($result->sizes as $key => $value) {
             if ($result->sizes[$key]->available == true) {
-                $dp[$result->sizes[$key]->slug] = $result->sizes[$key]->slug;
+                $dp[$tmp->slug] = $tmp->slug . '  ( $' . $tmp->price_monthly . ', RAM: ' . $tmp->memory . ')';
```

Comment:
Seems to me that $tmp is undefined, no?

Reply:
Not sure how that happened, but I made the change in my repo.  Do I need to do another PR?
`$tmp = $result->sizes[$key];`

Reply:
Nope, looks good to me

Reply:
Aweome, thanks!